src/Microsoft.Tye.Hosting/DockerRunner.cs

```diff
@@ -468,9 +468,15 @@ void Log(string data)
                         service.Logs.OnNext(data);
                     }
 
+                    var arguments = $"build \"{docker.DockerFileContext?.FullName}\" -t {dockerImage} -f \"{docker.DockerFile}\"";
```

Comment:
I'm still worried this won't give people the flexibility to add any arbitrary command. For example, there are a ton of [docker build options that can be passed in](https://docs.docker.com/engine/reference/commandline/build/#options) that can be passed in. Potentially supporting those may important as well outside of build-arg

Reply:
I would prefer to limit the scope of this change to build-arg only at this stage.  

I think if the other docker build options are name value pairs, this change should still work.  However, i take the point that i haven't tested anything other than build-args